lib/orphans.nix

```
@@ -0,0 +1,104 @@
+{ lib }:
+let
+  inherit (lib)
+    types
+    optional
+    isDerivation
+    attrByPath
+    flatten
+    any
+    assertMsg
+  ;
+  inherit (builtins)
+    length
+    concatStringsSep
+    mapAttrs
+    attrValues
+    tryEval
+  ;
+  isAttr = types.attrs.check;
+in
+rec {
+  /* Return a list of maintainers from a package, empty list if inaccessible.
+
+    Example:
+      listMaintainers pkgs.foo
+      => [ { ... } ]
+      listMaintainers pkgs.bar
+      => [ ]
+      listMaintainers {}
+      => error: listMaintainers must receive a package
```

Probably my mistake (I hate GitHub suggestion editor), sorry. But just to be clear:
- Paragraphs should be aligned with `/* `
- Indent everything inside "Example" with two spaces.

```suggestion
     Example:
       listMaintainers pkgs.foo
       => [ { ... } ]
       listMaintainers pkgs.bar
       => [ ]
       listMaintainers {}
       => error: listMaintainers must receive a package
```